Source/Charts/Renderers/AxisRendererBase.swift

```diff
@@ -97,7 +97,7 @@ open class AxisRendererBase: Renderer
         let labelCount = axis.labelCount
         let range = abs(yMax - yMin)
         
-        if labelCount == 0 || range <= 0 || range.isInfinite
+        if labelCount == 0 || range <= 0 || range.isInfinite || range.isNaN
```

Comment:
Why range is NaN in the first place? yMax and yMin should be guaranteed not inf nor NaN. So you should provide the reason.